### ISSUE TITLE
API Domain

### ISSUE THREAD

**Issue by davidhealey** [Daniel-Griffiths/vapour#24]
What domain name should I use when I request the API key?

**Comment by Daniel-Griffiths** [comment]
Hey @davidhealey 

No domain name needed, you can find your api key at: http://steamcommunity.com/dev/apikey

When the app asks you for it, enter that key and that should work. I have not updated Vapour in a really long time so there's a chance that it no longer works (eg if Valve have updated thier api).

**Comment by davidhealey** [comment]
Thanks for the quick response! I went to that link but it asks for a domain name. This is what I see (I suspect you might be right that the API has changed).

![image](https://user-images.githubusercontent.com/14021202/77650055-ca50ea00-6f62-11ea-86be-925afbb3534b.png)

**Comment by Daniel-Griffiths** [comment]
@davidhealey Hmmm, I don't recall that being an option so it looks like they added that for extra security. Is it possible to put `localhost` into the domain box?

**Comment by davidhealey** [comment]
Yeah that works. I can see my games now but clicking on them doesn't do anything. 

I think it might be something to do with these errors I'm seeing in the console.
![image](https://user-images.githubusercontent.com/14021202/77666863-64bc2800-6f79-11ea-8879-552c96fb5438.png)

I also noticed that when I first launched the app a webpage opened behind it with an error.
window.require is not a function

I'm also unable to sign in to Steam community
![image](https://user-images.githubusercontent.com/14021202/77667236-d6947180-6f79-11ea-979c-27a5d8f42c0a.png)
